src/cli/deck.py

```diff
@@ -234,6 +234,12 @@ def install(ctx, organization=None, project=None, deck=None, **kwargs):
     # check environment type
     check_environment_type_local_or_exit(deck=deck)
 
+    # check for switched app
+    provider_data = cluster.storage.get()
+    telepresence = Telepresence(provider_data)
+    if telepresence.intercept_count() > 0:
```

Comment:
@buschNT 
I just noticed that `.intercept_count` can return `None`. We should cover that as it fails in that case.
Can you make a PR with a little fix?